featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -117,33 +117,30 @@ def progress_callback(*args):
 
         if len(df.columns) == 0:
             default_df = self.generate_default_df(instance_ids=instance_ids)
-            if isinstance(df, dd.core.DataFrame):
+            if isinstance(df, dd.DataFrame):
                 cols = [col for col in default_df.columns] + [default_df.index.name]
                 return dd.from_pandas(default_df.reset_index(), npartitions=1)[cols]
             return default_df
 
         # Fill in empty rows with default values. This only works for pandas dataframes
         # and is not currently supported for Dask dataframes.
-        if isinstance(df, dd.core.DataFrame):
-            missing_ids = []
-        else:
+        if isinstance(df, pd.DataFrame):
             index_vals = df[target_entity.index].values
             missing_ids = [i for i in instance_ids if i not in index_vals]
```

Comment:
Can we also revert the code for creating `missing_ids` to match `master`?

`missing_ids = [i for i in instance_ids if i not in df[target_entity.index]]`